cinnamon/vizceral/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.6.3 
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    networks:
+      - cinnamon
+  backend:
+    image: lightbend-docker-registry.bintray.io/cinnamon/cinnamon-vizceral:0.1
+    ports:
+      - "8999:8999"
+    networks: 
+      - cinnamon
+    links:
+      - elasticsearch
```

Comment:
Are links needed, given the bridge network?

Reply:
Links are also used as `dependencies` IIRC. Can change to use `dependencies` instead?

Reply:
Yeah, `dependencies` could make more sense.